ur_robot_driver/doc/usage.rst

```diff
@@ -221,6 +223,7 @@ restarted again. Depending whether you use headless mode or not, you'll have to
 external_control program again.
 
 .. note::
+  Custom URScript command only works in headless mode. And does not work with the external_control URCap.
```

Comment:
```suggestion
  Custom URScript command only works in remote control mode on E-series robots or later.
```

Reply:
How about: "On E-series robots or later the robot needs to be in remote control mode in order to execute custom URScript commands."

Reply:
> "On E-series robots or *newer* the robot needs to be in remote control mode in order to execute custom URScript commands."

Reply:
```suggestion
  On E-series robots or newer the robot needs to be in remote control mode in order to execute custom URScript commands.
```